apps/web/src/pages/quick-start/components/OnboardingExperimentModal.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from 'react';
+import { Modal, useMantineTheme, Grid } from '@mantine/core';
+
+import styled from '@emotion/styled';
+import { colors, shadows, Title, Button } from '@novu/design-system';
+import { useAuthContext, useSegment } from '@novu/shared-web';
+import { useCreateOnboardingExperimentWorkflow } from '../../../api/hooks/notification-templates/useCreateOnboardingExperimentWorkflow';
+
+export function OnboardingExperimentModal() {
+  const [opened, setOpened] = useState(true);
+  const theme = useMantineTheme();
+  const segment = useSegment();
+  const { currentOrganization } = useAuthContext();
+  const {
+    createOnboardingExperimentWorkflow,
+    isLoading: IsCreateOnboardingExpWorkflowLoading,
+    isDisabled: isIsCreateOnboardingExpWorkflowDisabled,
+  } = useCreateOnboardingExperimentWorkflow();
+  const handleOnClose = () => {
+    setOpened(true);
+  };
+
+  return (
+    <Modal
+      opened={opened}
+      overlayColor={theme.colorScheme === 'dark' ? colors.BGDark : colors.BGLight}
+      overlayOpacity={0.7}
+      styles={{
+        modal: {
+          backgroundColor: theme.colorScheme === 'dark' ? colors.B15 : colors.white,
+          width: '700px',
+        },
+        body: {
+          paddingTop: '5px',
+          paddingInline: '8px',
+        },
+      }}
+      title={<Title size={2}>What would you like to do first?</Title>}
+      sx={{ backdropFilter: 'blur(10px)' }}
+      shadow={theme.colorScheme === 'dark' ? shadows.dark : shadows.medium}
+      radius="md"
+      size="lg"
+      onClose={handleOnClose}
+      centered
+      withCloseButton={false}
+    >
+      <Grid>
+        <Grid.Col xs={12} md={6} mb={20}>
+          <ChannelCard>
+            <TitleRow> {'Send test notification'}</TitleRow>
+            <Description>{'Learn how to setup a workflow and send your first email notification.'}</Description>
+            <StyledButton
+              loading={IsCreateOnboardingExpWorkflowLoading}
+              disabled={isIsCreateOnboardingExpWorkflowDisabled}
+              pulse={true}
+              variant={'gradient'}
+              onClick={async () => {
+                segment.track('Button Clicked - [Onboarding]', {
```

Comment:
please move this string to the `OnBoardingAnalyticsEnum`... same about the other places